Formula/fastani.rb

```diff
@@ -14,13 +14,14 @@ class Fastani < Formula
   end
 
   depends_on "autoconf" => :build
-  depends_on "gcc" => :build if OS.mac? # needs openmp
   depends_on "boost"
   depends_on "gsl"
 
   uses_from_macos "zlib"
 
-  fails_with :clang # needs openmp
+  on_macos do
+    depends_on "libomp" => :build
```

Comment:
There's no linkage to `libomp`? That's weird. Does it link statically? Are you certain that it's needed?

Reply:
This is an old commit; `fastani` does indeed link to `libomp`